lib/get-downloader.js

```diff
@@ -69,12 +68,12 @@ const getDownloader = ({
       logProcessing(remoteStatus.file);
       if (!command.force) {
         const localStatus = localStatusByFile[remoteStatus.file];
-        const { conflict } = getComparativeStatus(localStatus, remoteStatus);
+        const { conflict } = options.getComparativeStatus ? options.getComparativeStatus(localStatus, remoteStatus) : getComparativeStatus(localStatus, remoteStatus);
```

Comment:
hmmm I'm not sure we even need actual metafiles for data tables. Let's talk when you get in cause I have an idea. You could still getShallowRemoteStatus and getShallowLocalStatus, but they would be far simpler.